mobile-app/app/screens/AppNavigator/screens/Loans/LoansScreen.tsx

```diff
@@ -128,28 +131,52 @@ export function LoansScreen ({ route }: Props): JSX.Element {
 
   // TODO: remove custom handling of empty vault display
   useEffect(() => {
-    if (route.params?.displayEmptyVault === undefined) {
-      setDisplayEmptyVault(true)
+    if (route.params?.loadingState === undefined) {
+      setLoadingState('empty_vault')
     } else {
-      setDisplayEmptyVault(route.params.displayEmptyVault)
+      setLoadingState(route.params.loadingState)
     }
-  }, [route.params?.displayEmptyVault])
+  }, [route.params?.loadingState])
 
-  if (displayEmptyVault === true) {
+   // TODO: remove fake loading of loans
+  useEffect(
+    () => {
+      const loansTimer = setTimeout(() => {
+        console.log('here')
```

Comment:
Understand that it is temporary code, but still good to exclude any `console.log()`